src/config/texts.js

```diff
@@ -130,6 +130,12 @@ export const texts = {
     vote: 'Stimme',
     votes: 'Stimmen',
     startNew: {
+      'choose-document-content-type-application/msword-does-not-match-any-of-accepted-content-types-pdf':
+        'Inhaltstyp Document auswählen application/msword stimmt mit keinem der akzeptierten Inhaltstypen PDF überein.',
+      'choose-document-content-type-application/msword-does-not-match-any-of-accepted-content-types-pdf,-choose-document-must-be-in-between-0-bytes-and-3-mb':
```

Comment:
i guess that we need to work with a substring, as we cannot have all possible types represented here. so lets have `-does-not-match-any-of-accepted-content-types-pdf,-choose-document-must-be-in-between-0-bytes-and-3-mb` here and check if this is included when receiving validation errors. be careful when checking for include, because `-does-not-match-any-of-accepted-content-types-pdf` can match in two cases.